3d_segmentation/swin_unetr_brats21_segmentation_3d.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3D Brain Tumor Segmentation with Swin UNETR  (BraTS 21 Challenge)\n",
+    "\n",
+    "\n",
+    "This tutorial uses the [Swin UNETR](https://arxiv.org/pdf/2201.01266.pdf/) [1,2] model for the task of brain tumor  segmentation using the [BraTS 21](http://braintumorsegmentation.org/) challenge dataset [3,4,5,6]. Swin UNETR ranked among top-perfoming models in BraTS 21 validation phase. The architecture of Swin UNETR is demonstrated as below\n",
+    "![image](https://lh3.googleusercontent.com/j0NSTJl0_H6TBLvS4Wo_TTYpwk65OR0j1XmED6LA8T5Dg2MNH6dA-GZTaeoYMl8mDs1xamd-fyqjPgB0MRsvQhZv74jHOuOFEJFMbdY5q5vrOrfvUjUoHVepEHpx-hezpWxUJkv99ha_XVVl5j17JlNeJopDL4QqH6GFPJkBOI6OHOjjgGeYjOaLxIHcenVpdmPbfR3Xe1A_LsfLUqL6oLTJchORKi7xnjJRei6pLjZlXHgM_ZkesE1qXp_7tJ4CRjlwxaJCKW4JR3Zm6hgneiAuJL19fCq6_IGHoGDbLcYFA4oUuuyU6V-8HvF_ub5txiAdVpi3c4ez52ACfheUonZQycA5-haGnSkRaCUacPQHORr6ZCGyEtBmFe50uMlDg1lJ58MzUOGpdgHlSAvfqX_bWjYmhnnVF6h0WiXqmXRpVcYEjSTzRN3yMtD3r9lyxX3peDb1vla0685LwnKHTHmchM9o8_45C4tbiOOtRUrzHpcXaFXJgm1tNkTDpgM4J-41e3MjEuyO1W-H8w4T5eqpI-WhsKizqQzegXcemjW0mQz1zWORTfmdVgULKVy7trN0s4KLAfc-9um0qSGzHBXwRYp7NhpYMBVv_EdUvQJeC5V9WRaJOqzRM1myd9doOPi1ZXt-m_MJunXEzzff1HW7RHA-5x8IcmQUijsH_LA-Y1CZ6L7wVSY0jWHaB8Woaji0fotInv9lkE-nCJJsN28gWQbJ0TYL4DGDivja06Ea4E7sI2XfaIoWa0pfQHFDi-pC6EUy0LC-fdlIN1vqaY402wRuotKmjrOFBggJky_FeyncRkAyUYSvx7EU4Vs5_2P9o_woJvYUJnggX34JcnawyeFJVT9QSojWaBBeYHQTSoPgV5NZWh6tLSpbe2ZOLGEV6vQQJqPs2OKSZKXsOq3U5gNBYkORp_9xru0em9AONM5IRymRX99gCSYArM1ZDX6eBUL0fcdmDADPAFkH2dj82MeDAhdCTzV7Pg9iomfKWbjHrGixEWGrN0RYcGc2tz4Mhu6M72si9HpFXiy9L_c=w1388-h523-no?authuser=2)\n",
```

Comment:
Hi @ahatamiz , could you please double check the image link? I cannot see the figure here. Same issues in `Tumor Characteristics` and `Swin UNETR Model`

Reply:
Hi @yiheng-wang-nv 

Thanks for letting me know about this issue. I re-uploaded all three images to a publicly shared album. Would you please check again ? 

Thanks

Reply:
Thanks @ahatamiz , looks good to me.